game/view/startmenu.lua

```diff
@@ -29,21 +39,41 @@ local function _initFontValues()
   _width, _height = g.getDimensions()
 end
 
+local function _initLogo()
+  for i = 1, _LOGO_BG_PARTS do
+    _LOGO_BG[i] = RES.loadTexture('logo-bg'..i)
+    _LOGO_BG[i]:setFilter("linear","linear")
+    _LOGO_BG_OFFSET[i] = love.math.random(0,2*math.pi)
+    _LOGO_BG_MAGNITUDE[i] = love.math.random()*1 + 1 --Random float [1,2]
+  end
+  _LOGO_TEXT = RES.loadTexture('logo-text')
+  _LOGO_BG_WIDTH = _LOGO_BG[1]:getWidth()
+  _LOGO_BG_HEIGHT = _LOGO_BG[1]:getHeight()
+end
 
 local function _renderTitle(g)
   g.push()
-  g.translate(0, _height/4)
-  _title_font:set()
-  _title_font:setLineHeight(_LH)
+  g.translate(-_width/8, -_height/8)
   g.setColor(COLORS.NEUTRAL)
-  g.print(_TITLE_TEXT, 0, 0)
+  local x, y = 480,360
+  --Draw center without offset
+  g.draw(_LOGO_BG[1], x, y, _logo_rotation, nil, nil,
+        _LOGO_BG_OX, _LOGO_BG_OY)
+  --Draw all other parts with offset
+  for i = 2, _LOGO_BG_PARTS do
+    local offx = math.cos(_LOGO_BG_OFFSET[i])*_LOGO_BG_MAGNITUDE[i]
+    local offy = math.sin(_LOGO_BG_OFFSET[i])*_LOGO_BG_MAGNITUDE[i]
+    g.draw(_LOGO_BG[i],x+offx, y+offy, _logo_rotation, nil, nil,
+          _LOGO_BG_OX, _LOGO_BG_OY)
+  end
```

Comment:
i recommend moving the rendering code for the image to another function, and leave just the title here

Reply:
hmm ok